macros/edr/system/system_utils/get_config_var.sql

```diff
@@ -30,12 +38,17 @@
       'column_numeric': ['min', 'max', 'zero_count', 'zero_percent', 'average', 'standard_deviation', 'variance']
     },
     'time_format': '%Y-%m-%d %H:%M:%S',
-    'long_string_size': 16384,
+    'long_string_size': 65535,
     'collect_model_sql': true,
     'model_sql_max_size': 10240,
     'query_max_size': 1000000,
     'insert_rows_method': 'max_query_size'
   } %}
+  {{- return(default_config) -}}
+{%- endmacro -%}
 
-  {{ return(var(var_name, default_config.get(var_name))) }}
-{% endmacro %}
+{%- macro bigquery__get_default_config() -%}
+    {% set default_config = elementary.default__get_default_config() %}
```

Comment:
Not that important as we can't have the same dbt process run on two different adapters,
but you're overriding the default config for all adapters.

```suggestion
    {% set default_config = elementary.default__get_default_config().copy() %}
```

If we'd have logic that compares different config values among adapters, this would break it.

Reply:
I don't think that it's important, also don't want to pay the copy for every var access